src/app/pages/OnDemandAudioPage/index.test.tsx

```diff
@@ -486,4 +399,62 @@ describe('OnDemand Radio Page ', () => {
 
     expect(scheduleWrapper).not.toBeInTheDocument();
   });
+
+  describe('Page Identifier Overrides ', () => {
+    const afaanPageDataWithAvailableEpisode =
+      getAvailableEpisode(afaanoromooPageData);
+
+    it.each`
+      path                          | data                                 | language | pageIdentifierOverride                                   | pageTitleOverride          | service          | description
+      ${'some-podcast-radio-path'}  | ${persianPodcastBrandPageData}       | ${'fa'}  | ${'persian.bbc_persian_radio.podcasts.p0jrz542.page'}    | ${'پرگار'}                 | ${'persian'}     | ${'Persian Podcast'}
+      ${'some-ondemand-radio-path'} | ${indonesianBrandPageData}           | ${'id'}  | ${'indonesia.bbc_indonesian_radio.w172ywztppckjfb.page'} | ${'Dunia Pagi Ini'}        | ${'indonesia'}   | ${'Indonesian Brand'}
+      ${'some-ondemand-radio-path'} | ${persianDariEpisodePageData}        | ${'fa'}  | ${'persian.bbc_dari_radio.w3ct6lbh.page'}                | ${'مجله شامگاهی'}          | ${'persian'}     | ${'Persian Dari Episode'}
+      ${'some-ondemand-radio-path'} | ${persianEpisodePageData}            | ${'fa'}  | ${'persian.bbc_persian_radio.w3ct2czp.page'}             | ${'چشم انداز بامدادی'}     | ${'persian'}     | ${'Persian Episode'}
+      ${'some-ondemand-radio-path'} | ${afaanPageDataWithAvailableEpisode} | ${'om'}  | ${'afaanoromoo.bbc_afaanoromoo_radio.w3ct0l8r.page'}     | ${'Oduu BBC Afaan Oromoo'} | ${'afaanoromoo'} | ${'Afaan Oromoo Episode'}
+      ${'some-ondemand-radio-path'} | ${pashtoPageData}                    | ${'ps'}  | ${'pashto.bbc_pashto_radio.w3ct0lz1.page'}               | ${'ماښامنۍ خپرونه'}        | ${'pashto'}      | ${'Pashto Brand'}
+    `(
+      'should use the derived page identifier to render the audio player for $description',
+      async ({
+        path,
```

Comment:
Given that path is the same for each entry, should it be hardcoded and removed from the it.each section?

Reply:
I considered this but, I couldn't figure out a smart way to remove `path`
from the table because of this [podcast test](https://github.com/bbc/simorgh/pull/11998/files#diff-8df43a0a4619a5694eea5a7a9606f4aa1b49d3d8db2c98eeae505f321a5ada85R409) that uses a different path.
Shall we retain the table as is, on account of this?

Reply:
I don't think the path actually does anything, so I think we could hardcode it to something like `some-podcast-or-od-audio-path`. If that doesn't work we could leave it, but I would like to try to simplify this test as best we can

Reply:
The path is used to [determine](https://github.com/bbc/simorgh/blob/WSTEAMA-1382-remove-media-player-component-and-add-mediaLoaders/src/app/routes/onDemandAudio/getInitialData/index.ts#L28-L31) if the page requested is a Podcast page. The `isPodcast` value
in `pageData`, is then used to set the [pageIdentifierOverride](https://github.com/bbc/simorgh/blob/e3da7c55ec78eb20bda24a6385f33194c19034a7/src/app/pages/OnDemandAudioPage/OnDemandAudioPage.tsx#L141).
For this reason, we need it to assert that the correct `pageIdentifierOverride`
is inferred for On Demand Audio and Podcast pages.

Reply:
Oops I didn't realise. Ideally we would use the data to determine this (https://www.bbc.com/persian/podcasts/p02pc9mc.json contains version type), but let's leave it for now. Thanks @alex-magana